hax/hax/filestats.py

```diff
@@ -55,6 +55,11 @@ def _execute(self, motr: Motr):
                         not all(self.consul.ensure_ioservices_running()))):
                     wait_for_event(self.event, self.interval_sec)
                     continue
+                # Testing purpose and usage
```

Comment:
Do you mean this is a temporary code and a separate thread will be created to fetch bytecount right?

Reply:
Actually, wanted your suggestion here- will be it better to have a separate thread similar to fs-stats-updater thread? or use this same thread for fetching both stats?

Reply:
We need to have a separate thread similar to fsStatsUpdater, may be byteCountUpdater.

Reply:
Ok, it will be handled with task - https://jts.seagate.com/browse/EOS-27237 (Collect bytecount data from Motr and store in consul).